pkg/repository/crypto/rsa.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crypto
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/pem"
+)
+
+// RsaPair generate a pair of rsa keys
+func RsaPair() (*RsaPubKey, *RsaPriKey, error) {
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Put the key length to a const at top? We may need to change the value (or even support different values) in the future